Customer.API/Customer.API.csproj

```
@@ -7,11 +7,18 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.AspNet.WebApi.Core" Version="5.2.9" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore.Design" Version="6.0.9">
+      <PrivateAssets>all</PrivateAssets>
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+    </PackageReference>
+    <PackageReference Include="Microsoft.VisualStudio.Azure.Containers.Tools.Targets" Version="1.17.0" />
     <PackageReference Include="Swashbuckle.AspNetCore" Version="6.2.3" />
   </ItemGroup>
 
   <ItemGroup>
     <ProjectReference Include="..\Customer.AppServices\Customer.AppServices.csproj" />
+    <ProjectReference Include="..\Customer.Infrastructure.Data\Customer.Infrastructure.Data.csproj" />
   </ItemGroup>
```

A API não pode ter acesso à camada de Data
```suggestion
  <ItemGroup>
    <ProjectReference Include="..\Customer.AppServices\Customer.AppServices.csproj" />
  </ItemGroup>
```